### PR TITLE
[Merged by Bors] - feat: port Order.ConditionallyCompleteLattice.Finset

### PR DESCRIPTION
**Pull request by ChrisHughes24**
---
easy

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Order/ConditionallyCompleteLattice/Finset.lean [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+#align set.finite.cSup_lt_iff Set.Finite.cSup_lt_iff
+
+theorem Set.Finite.lt_cInf_iff (hs : s.Finite) (h : s.Nonempty) : a < infₛ s ↔ ∀ x ∈ s, a < x :=
+  @Set.Finite.cSup_lt_iff αᵒᵈ _ _ _ hs h
+#align set.finite.lt_cInf_iff Set.Finite.lt_cInf_iff
+
+end ConditionallyCompleteLinearOrder
+
+/-!
+### Relation between `Sup` / `Inf` and `finset.sup'` / `finset.inf'`
```
Comment (jcommelin) [inline Mathlib/Order/ConditionallyCompleteLattice/Finset.lean:81]:
```suggestion
### Relation between `Sup` / `Inf` and `Finset.sup'` / `Finset.inf'`
```

File: Mathlib/Order/ConditionallyCompleteLattice/Finset.lean [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+Like the `Sup` of a `conditionally_complete_lattice`, `finset.sup'` also requires the set to be
```
Comment (jcommelin) [inline Mathlib/Order/ConditionallyCompleteLattice/Finset.lean:83]:
```suggestion
Like the `Sup` of a `ConditionallyCompleteLattice`, `Finset.sup'` also requires the set to be
```

**Review by jcommelin — COMMENTED** [review]
bors d+

**Comment by jcommelin** [comment]
bors merge

**Comment by ChrisHughes24** [comment]
bors merge

**Pull request closed without merging**